composer.lock

```diff
@@ -838,40 +838,43 @@
         },
         {
             "name": "elasticsearch/elasticsearch",
-            "version": "v6.7.2",
```

Comment:
Not sure whats requiring this but this broke the pipeline a week ago https://phabricator.wikimedia.org/T276320